torch/cuda/__init__.py

```diff
@@ -898,18 +898,18 @@ def _dtype(self):
 del _LegacyStorage
 del _CudaLegacyStorage
 
-torch._storage_classes.add(DoubleStorage)
-torch._storage_classes.add(FloatStorage)
-torch._storage_classes.add(LongStorage)
-torch._storage_classes.add(IntStorage)
-torch._storage_classes.add(ShortStorage)
-torch._storage_classes.add(CharStorage)
-torch._storage_classes.add(ByteStorage)
-torch._storage_classes.add(HalfStorage)
-torch._storage_classes.add(BoolStorage)
-torch._storage_classes.add(BFloat16Storage)
-torch._storage_classes.add(ComplexDoubleStorage)
-torch._storage_classes.add(ComplexFloatStorage)
+torch._storage_classes.add(DoubleStorage)  # type: ignore[has-type]
+torch._storage_classes.add(FloatStorage)  # type: ignore[has-type]
+torch._storage_classes.add(LongStorage)  # type: ignore[has-type]
+torch._storage_classes.add(IntStorage)  # type: ignore[has-type]
+torch._storage_classes.add(ShortStorage)  # type: ignore[has-type]
+torch._storage_classes.add(CharStorage)  # type: ignore[has-type]
+torch._storage_classes.add(ByteStorage)  # type: ignore[has-type]
+torch._storage_classes.add(HalfStorage)  # type: ignore[has-type]
+torch._storage_classes.add(BoolStorage)  # type: ignore[has-type]
+torch._storage_classes.add(BFloat16Storage)  # type: ignore[has-type]
+torch._storage_classes.add(ComplexDoubleStorage)  # type: ignore[has-type]
+torch._storage_classes.add(ComplexFloatStorage)  # type: ignore[has-type]
```

Comment:
I'm really confused, why this change was necessary?

Reply:
@amjames , could it be there is a leakage coming from the lazy initialization process?

Reply:
And now we have even more coming our way...